main.go

```diff
@@ -133,13 +133,25 @@ func main() {
 		os.Exit(1)
 	}
 
+	podNamespace, found := os.LookupEnv("POD_NAMESPACE")
+	if !found {
+		podNamespace = config.GetDefaultMCONamespace()
+	}
+
+	// set image manifests configmap name. e.g.: mch-image-manifest-2.3.0
+	config.SetImageManifestConfigMapName()
+
 	gvkLabelMap := map[schema.GroupVersionKind]filteredcache.Selector{
 		v1.SchemeGroupVersion.WithKind("Secret"): {
 			FieldSelector: fmt.Sprintf("metadata.namespace==%s", config.GetDefaultNamespace()),
 		},
 		v1.SchemeGroupVersion.WithKind("ConfigMap"): {
 			FieldSelector: fmt.Sprintf("metadata.namespace==%s", config.GetDefaultNamespace()),
 		},
+		v1.SchemeGroupVersion.WithKind("ConfigMap"): {
```

Comment:
can we have two entities for the same resource kind in the Selector?

Reply:
That works in my testing. @morvencao thoughts?

Reply:
It should not work.
We will lose the first `configmap` setting.
Did you verify the configmap in `open-cluster-management-observability` is still be watching?